.circleci/config.yml

```
@@ -44,7 +44,7 @@ workflows:
                   docker run \
                       -it --name app-test --rm \
                       --link=database \
-                      --env-file=dev.env \
+                      --env-file=.circleci/circleci.env \
```

We've done this some places before, and just referenced the example file (and overridden anything "wrong" in it later, like the DB uri) -- it seems like this should work fine here too, and saves us keeping a copy
```suggestion
                      --env-file=dev.env.example \
```